### ISSUE TITLE
[Discussion] How many braced variables should be put on one line?

### ISSUE THREAD

**Issue by aDogCalledSpot** [schvv31n/yew-fmt#15]
It makes sense to say that 

```rust
html! {
    <Component>{ foo }</Component>
```

is  formatted correctly.

However something like

```rust
html! {
    <Component>
         { foo }
         { bar }
         { baz }
         { foobar }
   </Component>
}
```

should maybe be kept that way even though it could possibly be merged onto one line.

I think there might be semantic meaning to the variables that would make you prefer splitting the lines (e.g. components into new lines, strings onto same line).

Would always splitting lines at two variables make sense?

**Comment by schvv31n** [comment]
My idea with leaving multiple blocks on 1 line was that this way the blocks will look like string interpolation
```html
html! {
    <h1>{ "Hello, " }{ user }</h1>
}
```
But I agree there might be cases where that's not desired, maybe there should be a formatting option for that

**Comment by aDogCalledSpot** [comment]
Yes, that's what I was thinking of too. I think there is semantic meaning that  decides how a user would want to format this which is problematic for a formatter :sweat_smile:
